server/db/seed.js

```diff
@@ -0,0 +1,49 @@
+const db = require('./index.js');
+const Graph = require('./graph.js');
+
+const companyNames = require('./templates/companyNames');
+const dataCreate = require('./templates/dataCreate');
+
+const createObjects = () => {
+  let currentStockPrice = Math.floor(Math.random() * 100);
+  let graphData  = [];
+
+  // integer that is the current price
+  console.log("creating new collections...")
+  Object.entries(companyNames).forEach( (entry, idx) => {
+    let newGraph = {
+      company: entry[0],
+      title: entry[1],
+      scale: "",
+      data: dataCreate(currentStockPrice)
+    }
+
+    console.log(`Companies: ${idx + 1}/${Object.keys(companyNames).length}`)
```

Comment:
I really like how you use console.logs to keep track of where you're at in the seeding process